packages/api/src/middleware/hardcoded-nodes.ts

```diff
@@ -41,8 +41,10 @@ export default function hardcodedNodes({
     if (!req.getBroadcasters) {
       req.getBroadcasters = async () => broadcasters as NodeAddress[];
     }
-    if (!req.getOrchestrators) {
-      req.getOrchestrators = async () => orchestrators as NodeAddress[];
+    if (orchestrators.length) {
```

Comment:
If the `orchestrators` might ever be `null/undefined`, might be good to use a coalesce `?.` here
```suggestion
    if (orchestrators?.length) {
```

Reply:
fixed